test/cypress/stubs/gateway-account-stubs.js

```diff
@@ -196,6 +196,14 @@ function postCheckWorldpay3dsFlexCredentials (opts) {
   }
 }
 
+function postCheckWorldpayCredentials (opts) {
+  const path = `/v1/api/accounts/${opts.gatewayAccountId}/worldpay/check-credentials`
+  return stubBuilder('POST', path, 200, {
+    request: opts.credentials,
+    response: { result: 'valid' }
```

Comment:
Should use a fixture with a corresponding pact test. Fine for that to be in a separate PR though.